servicetalk-concurrent-api/src/test/java/io/servicetalk/concurrent/api/PublisherConcatMapIterableTest.java

```diff
@@ -315,6 +316,7 @@ public void exceptionFromBufferedOnNextThenOnErrorIsPropagated() {
         testExceptionFromBufferedOnNextThenTerminalIsPropagated(publisher::onError);
     }
 
+    @Ignore("fixed in https://github.com/apple/servicetalk/pull/1229")
```

Comment:
the updated test are in https://github.com/apple/servicetalk/pull/1229 and depend upon a few core changes (e.g. `TestPublisher`, `TestPublisherSubscriber`) so I will defer the testing to that PR with the larger changes.